src/domain/contracts/entities/contract.entity.ts

```diff
@@ -1,8 +1,4 @@
-export interface Contract {
-  address: string;
-  name: string;
-  displayName: string;
-  logoUri: string;
```

Comment:
I checked the Transaction Service and this can return `null` which has been reflected in the new `ContractSchema`.

Reply:
Good catch! In fact, in the schema it was considered nullable already.